tests/test_app.py

```diff
@@ -1,2 +1,2 @@
 def test_app():
-	pass
\ No newline at end of file
+    pass
```

Comment:
you might need something like this - https://github.com/openclimatefix/pv-site-production/blob/main/forecast-inference/tests/conftest.py#L41